policybot/pkg/syncer/syncer.go

```diff
@@ -30,8 +30,11 @@ import (
 	"google.golang.org/api/iterator"
 	"google.golang.org/api/option"
 
+	gcs "cloud.google.com/go/storage"
```

Comment:
Let's use the blob storage client rather than GCS directly. See pkg/blobstorage.

Thanks.